tools.go

```diff
@@ -5,9 +5,17 @@ package eksctl
 // Mock imports to enforce their installation by `go mod`.
 // See https://github.com/golang/go/wiki/Modules#how-can-i-track-tool-dependencies-for-a-module
 import (
+	// This go-version dependency only included to enforce aws-iam-authenticator to use a specific version of go-version.
+	// aws-iam-authenticator v0.4.0 uses `dep`, causing `go mod` to ignore its dependency files and just to pull
+	// github.com/christopherhein/go-version:master. This causes the build to break because go-version:master has recently
+	// switched to module name go.hein.dev/go-version, causing a module name mismatch.
+	// TODO(fons): get rid of this dependency once we bump the version of aws-iam-authenticator
+	//             which now uses go modules
```

Comment:
@errordeveloper FYI, the fact that this is possible makes it obvious that you cannot depend on multiple versions of a library (enforcing a version of a transitive dependency also changes the version of such dependency)

Reply:
👍